### PR TITLE
RDP fixes

### PR DESCRIPTION
**Pull request by d3agle**

### CODE REVIEW

File: Server/Controls/RapidPictureBox.cs [hunk 1]
```diff
@@ -92,8 +92,6 @@ public void UnsetFrameUpdatedEvent(FrameUpdatedEventHandler e)
         /// </summary>
         public void Start()
         {
-            _frameCounter = new FrameCounter();
```
Comment (MaxXor) [inline Server/Controls/RapidPictureBox.cs]: Why is this removed??

File: Server/Forms/FrmRemoteDesktop.cs [hunk 2]
```diff
@@ -50,31 +46,36 @@ private void FrmRemoteDesktop_Load(object sender, EventArgs e)
...
+                _keyboardHook.KeyUp -= OnKeyUp;
+                _mouseHook.MouseWheel -= OnMouseWheelMove;
+
+                _mouseHook.Dispose();
+                _keyboardHook.Dispose();
+            }
 
-        private void UnsubscribeMonoEvents()
-        {
             this.KeyDown -= OnKeyDown;
```
Comment (MaxXor) [inline Server/Forms/FrmRemoteDesktop.cs]: Add a check for Mono platform, like in subscribe

File: Server/Controls/RapidPictureBox.cs (hunk 1 above)
Comment (d3agle) [inline Server/Controls/RapidPictureBox.cs]: Because it is created in the constructor

Comment (MaxXor) [inline Server/Controls/RapidPictureBox.cs]: Well, okay. I can live with that. But the FPS won't be accurate for the first few seconds when stopping and starting remote desktop again.^^

Comment (d3agle) [inline Server/Controls/RapidPictureBox.cs]: True, I will leave it in the Start method then, and remove from constructor :P

Comment (MaxXor) [inline Server/Controls/RapidPictureBox.cs]: Yes, that's fine.
